backend-truffle/contracts/DoorManager.sol

```diff
@@ -1,77 +1,61 @@
-//Meet.inc >>> ĐOor-Manager <<<
-//The ĐOor-Manager is a factory that allows event managers to deploy their own Door Event.
-//ĐOor opens doors at events that are anchored on the Ethereum blockchain.
-//
-//Last update: 10.11.2019
-
 pragma solidity ^0.5.0;
 
 import "@openzeppelin/contracts/ownership/Ownable.sol";
 import "@openzeppelin/contracts/lifecycle/Pausable.sol";
-import "./DoorEvent.sol";
+import "./Door.sol";
 
+contract DoorFactory is Ownable, Pausable {
 
-contract DoorManager is Ownable, Pausable{
-    //Variables
-    address[] private doorEventAddresses;
-    mapping (address => bool) private doorEventCreated;
+    address[] private doors;
+    mapping (address => bool) private createdConfirmations;
 
-    //Events
-    event NewDoorCreated(
-        address indexed doorOwner,
+    event Created(
```

Comment:
question is -> if the factory already emits these events... why should the door contract be able to do so? because it should be possible that someone spawns a door without using the factory? then you don't need to emit these events from the factory but solely can trust the spawned door contracts to emit them...

Reply:
at the moment the DoorEvent.sol is not emitting a "door creation" event. You think we do not need that?
DoorEvent.sol is just emitting an event called "LogDoorEventTicketSold(msg.sender)" where we can listen on ("ticket counter" or "X tickets sold - hurry up")